packages/zipkin/src/tracer/index.js

```diff
@@ -69,6 +69,23 @@ class Tracer {
     return childId;
   }
 
+  // creates a span, timing the given callable, adding any error as a tag
+  local(operationName, callable) {
```

Comment:
For the check we could either be explicit and throw an error directly, but this might be too disruptive for our users. I would go for a check if it's a function and if not simply return the callable